tests/bluetooth/bsim/nrf_auraconfig/tests/usecase_0_test.sh

```diff
@@ -9,6 +9,34 @@ SCRIPT_DIR=$(dirname "$(realpath "${BASH_SOURCE[0]}")")
 SCRIPT_NAME=$(basename "$0")
 
 ${SCRIPT_DIR}/../_nrf_auraconfig_simulation.sh ${SCRIPT_NAME} \
-  "nac" "usecase" "0"\
+  "nac" "preset" "24_2_1" "0"\
+  "nac" "adv_name" "'Lecture hall'" "0"\
+  "nac" "broadcast_name" "Lecture" "0"\
+  "nac" "broadcast_id" "fixed" "0" "0x123456"\
+  "nac" "packing" "int" "0"\
+  "nac" "lang" "eng" "0" "0"\
+  "nac" "context" "live" "0" "0"\
+  "nac" "program_info" "'Mathematics 101'" "0" "0"\
+  "nac" "immediate" "1" "0" "0"\
+  "nac" "num_bises" "1" "0" "0"\
   "nac" "start" "0"\
-  "nac_test" "test_uc0" \
+  "nac_test" "adv_name" "'Lecture hall'"\
+  "nac_test" "broadcast_name" "Lecture"\
+  "nac_test" "broadcast_id" "0x123456"\
+  "nac_test" "phy" "2"\
+  "nac_test" "pd_us" "40000"\
+  "nac_test" "encryption" "0"\
+  "nac_test" "broadcast_code" "blank"\
+  "nac_test" "num_subgroups" "1"\
+  "nac_test" "std_quality" "1"\
+  "nac_test" "high_quality" "0"\
+  "nac_test" "sampling_rate" "0" "24000"\
+  "nac_test" "frame_duration" "0" "10000"\
+  "nac_test" "octets_per_frame" "0" "60"\
+  "nac_test" "language" "0" "eng"\
+  "nac_test" "immediate_rend_flag" "0" "1"\
+  "nac_test" "context" "0" "live"\
+  "nac_test" "program_info" "0" "'Mathematics 101'"\
+  "nac_test" "num_bis" "0" "1"\
+  "nac_test" "bis_location" "0" "0" "0x00"\
+  "nac_test" "run_test"\
```

Comment:
Could you have an include file that has a set of human readable constants to fill these in with?
For example have a usecase_defines.sh with constants defined as:
declare -i BIG_0=0
declare -i BIG_1=1

Then include it in each .sh file with: source usecase_defines.sh

Then use as: "nac_test" "frame_duration" "BIG_0" "FRM_DUR_10000_US"\
This would make it a bit more readable.

Reply:
Yeah, probably could do that, will look into it